string node_name
string new_name
---
bool success